override/aws/imdsretryer.go

```diff
@@ -33,28 +34,28 @@ import (
 var IMDSRetryer request.Retryer = newIMDSRetryer()
 
 const (
-	TimePerCall = 2 * time.Minute
+	TimePerCall = 30 * time.Second
```

Comment:
What is the context for reducing this TimePerCall?
Business decision?

Reply:
We had a flaky test for 2 minutes in the private. I changed it to 30 seconds there so I changed it to 30 seconds here to match.